src/hook/useScrollTo.ts

```diff
@@ -0,0 +1,27 @@
+import { useLayoutEffect } from 'react';
+import useSessionStorage from './useSessionStorage';
+
+interface useScrollToProps {
+  key: string;
+}
```

Comment:
```suggestion
interface UseScrollToProps {
  key: string;
}
```
파스칼 케이스로 변경하는 게 좋아보입니당